bittensor/_neuron/text/core_server/nucleus_impl.py

```diff
@@ -205,6 +205,15 @@ def remapping_token(self, token_batch, std_tokenizer=None, return_offsets_mappin
             std_tokenizer = self.std_tokenizer
 
         text_batch = std_tokenizer.batch_decode(token_batch)  # decode tokens to original text
+
+        # displaying a preview of the inference data
+        print(('Inference Data: ' + str(text_batch)[2:65]) + '......' + (str(text_batch)[-65:-2]))
```

Comment:
This whole feature should probably be a flag, and disabled by default, rather than enabled always.